src/Compilers/CSharp/Test/Emit/Attributes/InternalsVisibleToAndStrongNameTests.cs

```diff
@@ -1016,6 +1016,75 @@ public class C
         #region Signing
 
         [Fact]
+        public void MaxSizeKey()
+        {
+            var pubKey = TestResources.General.snMaxSizePublicKeyString;
+            const string pubKeyToken = "1540923db30520b2";
```

Comment:
>pubKeyToken [](start = 25, length = 11)

Is this a string representation of ```pubKeyTokenBytes```? It feels strange that we are hardcoding the string, but not the bytes. Perhaps should do the same for both?